requirements/test.txt

```diff
@@ -5,24 +5,19 @@
 #    make upgrade
 #
 astroid==1.6.6            # via pylint
-backports.functools-lru-cache==1.6.1  # via astroid, isort, pylint
-configparser==4.0.2       # via pylint
-coverage==5.2.1           # via -r requirements/test.in
+coverage==5.3             # via -r requirements/test.in
 django-crum==0.7.7        # via -r requirements/base.txt
 django-ipware==2.1.0      # via -c requirements/constraints.txt, -r requirements/base.txt
-django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.txt, django-crum, jsonfield
-enum34==1.1.10            # via astroid
-funcsigs==1.0.2           # via mock
-futures==3.3.0 ; python_version == "2.7"  # via -c requirements/constraints.txt, isort
+django==1.11.29           # via -c requirements/constraints.txt, -r requirements/base.txt, django-crum, jsonfield2
 isort==4.3.21             # via pylint
-jsonfield==2.0.2          # via -c requirements/constraints.txt, -r requirements/base.txt
+jsonfield2==3.0.3 ; python_version > "3.5"  # via -c requirements/constraints.txt, -r requirements/test.in
```

Comment:
did you add this manually ?? or the next line

Reply:
I had to add manually jsonfield==2.0.2 ; python_version == "2.7" since it was not added with the make upgrade command. I believe it has to do with the python version of my environment where I ran the upgrade?

Reply:
I had to add manually jsonfield==2.0.2 ; python_version == "2.7" since it was not generated when I run the make upgrade command. I believe it has to do with the python version of my environment where I run the upgrade?